.cirrus.yml

```diff
@@ -86,9 +86,6 @@ task:
         - dart pub run test
     - name: publishable
       env:
-        # TODO (mvanbeusekom): Temporary override to "stable" because of failure on "master".
-        # Remove override once https://github.com/dart-lang/pub/issues/3152 is resolved.
-        CHANNEL: stable
```

Comment:
This is opportunistic cleanup since it came up in my sweep of every reference to "master"; the issue was resolved quite a while ago.